packages/main/src/components/protected/sidebar/components/Sidebar.jsx

```diff
@@ -25,7 +25,7 @@ const Sidebar = props => {
 
   const sidebarRef = useRef(null);
   const [isResizing, setIsResizing] = useState(false);
-  const [sidebarWidth, setSidebarWidth] = useState(150);
+  const [sidebarWidth, setSidebarWidth] = useState(240);
```

Comment:
Did you pull, I don't think you need to touch the sidebar again. @JulianaSau

Reply:
@willhemz please don't touch the sidebar🥲. We just fixed it

Reply:
Alright. I'll do that now. Thanks

Reply:
@billmal071 @JulianaSau my branch is updated with recent changes. Kindly re-review. thanks

Reply:
@JulianaSau @billmal071 I've added the functionality, such that members already in the channel will not display in the search option.